core/domain/stats_domain_test.py

```diff
@@ -385,6 +510,54 @@ def test_validate(self):
             'Expected exp_id to be a string, received %s' % (type(5)))):
             exp_issues.validate()
 
+    def test_validate_exp_version(self):
+        exp_issues = stats_domain.ExplorationIssues(
+            'exp_id1', 1, [
+                stats_domain.ExplorationIssue.from_dict({
+                    'issue_type': 'EarlyQuit',
+                    'issue_customization_args': {
+                        'state_name': {
+                            'value': 'state_name1'
+                        },
+                        'time_spent_in_exp_in_msecs': {
+                            'value': 200
+                        }
+                    },
+                    'playthrough_ids': ['playthrough_id1'],
+                    'schema_version': 1,
+                    'is_valid': True})
+                ])
+        exp_issues.validate()
+
+        exp_issues.exp_version = 'invalid_version'
```

Comment:
nit: add line to separate "prepare, act, assert" logic here and elsewhere.

Reply:
Done!